.github/workflows/deploy.yml

```diff
@@ -267,4 +267,13 @@ jobs:
         uses: icaraps/gatsby-fastly-purge-action@master
         with:
           fastly-token: ${{ secrets.AIO_FASTLY_TOKEN }}
-          fastly-url: '${{ secrets.AIO_FASTLY_PROD_URL }}${{ needs.set-state.outputs.path_prefix }}'
\ No newline at end of file
+          fastly-url: '${{ secrets.AIO_FASTLY_PROD_URL }}${{ needs.set-state.outputs.path_prefix }}'
+      - name: Create GitHub release tag
+        id: create_release
+        uses: release-drafter/release-drafter@v5
+        with:
+          name: Deployment ${{ github.run_id }}
+          tag: ${{ github.run_id }}
```

Comment:
Can you prefix it with date in YYYYMMDD so that it makes it easy when searching for tags?

Reply:
Updated it to include local date in the name and tag. For tag name date is a suffix, since it was not able to successfully compare and create the tags with the prefix. Let me know what you think of this example:
![Screenshot 2023-09-15 at 6 23 50 PM](https://github.com/AdobeDocs/aep-mobile-sdkdocs/assets/33132425/1c5e568c-1c4f-4ffc-82e9-65e03a4e7bbd)

![Screenshot 2023-09-15 at 6 19 44 PM](https://github.com/AdobeDocs/aep-mobile-sdkdocs/assets/33132425/111f18f0-dd33-4ac5-b89b-74250051f305)